packages/@aws-cdk/cloudformation-diff/lib/iam/iam-changes.ts

```diff
@@ -71,21 +71,13 @@ export class IamChanges {
     for (const statement of this.statements.additions) {
       ret.push([
         '+',
-        renderTargets(statement.resources),
-        statement.effect,
-        renderTargets(statement.actions),
-        renderTargets(statement.principals),
-        renderCondition(statement.condition),
+        ...statement.render(),
```

Comment:
I mean... yeah... but the concern about the table and what columns there were in what order USED to be in `summarizeStatements()` and now it's split across `summarizeStatements()` and `statement.render()`.

`statement.render()` should return an object and `summarizeStatements()` should pick fields out of that object in the right order.

Reply:
Done!